### PR TITLE
Add blog post reflecting on CollabW21

### PR DESCRIPTION
**Pull request by sgibson91**
This PR is

- Adding Sarah to the list of authors
- Adding a new blog post reflecting on CollabW21

### CODE REVIEW

File: _posts/blog/2021-04-08-whitakerlab-at-collabw21.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+image:
+  feature:
+link:
+date: 2021-04-08 ### UPDATE date
+modified:
+share: true
+author: sarah_gibson ### CHANGE to your author name (in _data/authors.yml)
+---
+
+At the end of March 2021, meny member's of the Whitaker Lab attended the Collaborations Workshop hosted by the Software Sustainability Institute and, at the following lab meeting, those who attended shared their experiences with the rest of the lab.
```
Comment (BrainonSilicon) [inline _posts/blog/2021-04-08-whitakerlab-at-collabw21.md]: Spelling adjustment: 
```spelling and grammar adjustment
At the end of March 2021, many members of the Whitaker Lab attended the Software Sustainability Institute's Collaborations Workshop. At the following lab meeting, those who attended shared their experiences with the rest of the lab.
```

File: _posts/blog/2021-04-08-whitakerlab-at-collabw21.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+The [Software Sustainability Institute](https://software.ac.uk/about) (SSI for short) is a network of UK universities, Research Software Engineering groups and policy makers dedicated in improving the quality, sustainability and recognition of research software.
+They help the members in their network learn software skills and best practices, and advocate for culture change in their organisations and institutions.
+
+Many members of the lab (current and past) are also [SSI Fellows](https://software.ac.uk/programmes-and-events/fellowship-programme).
+[Can you spot them?](https://software.ac.uk/about/fellows)
+
+## What is Collaborations Workshop?
+
+The [Collaborations Workshop](https://software.ac.uk/programmes-and-events/collaborations-workshops) (often shorted to CollabW) is an annual, 3-day event focussed around bringing together researchers, software developers, data librarians, managers, funders and anyone else who cares about research software to explore important ideas around software and research and foster interdisciplinary collaborations.
```
Comment (BrainonSilicon) [inline _posts/blog/2021-04-08-whitakerlab-at-collabw21.md]:
```spelling and grammar suggestion
The [Collaborations Workshop](https://software.ac.uk/programmes-and-events/collaborations-workshops) (often shorted to CollabW) is an annual, 3-day event focused around bringing together researchers, software developers, data librarians, managers, funders and anyone else who cares about research software to foster interdisciplinary collaborations and explore important ideas around software and research. 
```

**Review by BrainonSilicon — CHANGES_REQUESTED** [review]
Adding a few phrasing suggestions and did a pass over spelling :)

File: _posts/blog/2021-04-08-whitakerlab-at-collabw21.md (hunk 1 above)
Comment (sgibson91) [reply _posts/blog/2021-04-08-whitakerlab-at-collabw21.md]:
```suggestion
At the end of March 2021, many members of the Whitaker Lab attended the Software Sustainability Institute's Collaborations Workshop.
At the following lab meeting, those who attended shared their experiences with the rest of the lab.
```

File: _posts/blog/2021-04-08-whitakerlab-at-collabw21.md (hunk 2 above)
Comment (sgibson91) [reply _posts/blog/2021-04-08-whitakerlab-at-collabw21.md]:
```suggestion
The [Collaborations Workshop](https://software.ac.uk/programmes-and-events/collaborations-workshops) (often shorted to CollabW) is an annual, 3-day event focused around bringing together researchers, software developers, data librarians, managers, funders and anyone else who cares about research software to foster interdisciplinary collaborations and explore important ideas around software and research.
```

File: _posts/blog/2021-04-08-whitakerlab-at-collabw21.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+link:
+date: 2021-04-08 ### UPDATE date
+modified:
+share: true
+author: sarah_gibson ### CHANGE to your author name (in _data/authors.yml)
+---
+
+At the end of March 2021, many members of the Whitaker Lab attended the Software Sustainability Institute's Collaborations Workshop.
+At the following lab meeting, those who attended shared their experiences with the rest of the lab.
+This is a blog post covering the event and the lab member's reflections from it.
```
Comment (MariaEriksson) [inline _posts/blog/2021-04-08-whitakerlab-at-collabw21.md]:
```suggestion
This is a blog post covering the event and the lab members' reflections from it.
```

File: _posts/blog/2021-04-08-whitakerlab-at-collabw21.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+## What is the SSI?
+
+The [Software Sustainability Institute](https://software.ac.uk/about) (SSI for short) is a network of UK universities, Research Software Engineering groups and policy makers dedicated in improving the quality, sustainability and recognition of research software.
```
Comment (MariaEriksson) [inline _posts/blog/2021-04-08-whitakerlab-at-collabw21.md]: I might have said "dedicated _to_", but perhaps that's more of a stylistic thing?

File: _posts/blog/2021-04-08-whitakerlab-at-collabw21.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+They help the members in their network learn software skills and best practices, and advocate for culture change in their organisations and institutions.
+
+Many members of the lab (current and past) are also [SSI Fellows](https://software.ac.uk/programmes-and-events/fellowship-programme).
+[Can you spot them?](https://software.ac.uk/about/fellows)
+
+## What is Collaborations Workshop?
```
Comment (MariaEriksson) [inline _posts/blog/2021-04-08-whitakerlab-at-collabw21.md]:
```suggestion
## What is the Collaborations Workshop?
```

File: _posts/blog/2021-04-08-whitakerlab-at-collabw21.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+
+The [Collaborations Workshop](https://software.ac.uk/programmes-and-events/collaborations-workshops) (often shorted to CollabW) is an annual, 3-day event focused around bringing together researchers, software developers, data librarians, managers, funders and anyone else who cares about research software to foster interdisciplinary collaborations and explore important ideas around software and research.
```
Comment (MariaEriksson) [inline _posts/blog/2021-04-08-whitakerlab-at-collabw21.md]:
```suggestion
The [Collaborations Workshop](https://software.ac.uk/programmes-and-events/collaborations-workshops) (often shortened to CollabW) is an annual, 3-day event focused around bringing together researchers, software developers, data librarians, managers, funders and anyone else who cares about research software to foster interdisciplinary collaborations and explore important ideas around software and research.
```

File: _posts/blog/2021-04-08-whitakerlab-at-collabw21.md [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+
+CollabW features a few keynotes but it is largely run as an [unconference event](https://unconference.net/unconferencing-how-to-prepare-to-attend-an-unconference/) with lightning talks, discussion sessions, mini workshops, and speed blogging events all relating to one or a few central themes.
+This feels radically different to traditional conference events as you are elevated from audience member to participant, giving you the opportunity to create something during the event and have discussions with a wide variety of other attendees.
```
Comment (MariaEriksson) [inline _posts/blog/2021-04-08-whitakerlab-at-collabw21.md:33]: I really like how you describe the event - makes me want to go next time! 😄

File: _posts/blog/2021-04-08-whitakerlab-at-collabw21.md [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+
+The final day features a [hack day](https://software.ac.uk/cw21/hack-day) where small teams can work on community-generated projects addressing a need they see in the research software community.
```
Comment (MariaEriksson) [inline _posts/blog/2021-04-08-whitakerlab-at-collabw21.md:35]: Would you be able to give just a few examples of the community-generated projects here?

File: _posts/blog/2021-04-08-whitakerlab-at-collabw21.md [hunk 9]
```diff
@@ -0,0 +1,71 @@
...
+Importantly, these do not have to be software projects to be eligible.
+Documentation and event planning projects, among others, have also featured at previous hack days.
+But most important of all, there are prizes up for grabs for the top 3 projects! :wink:
+
+## CollabW21
+
+The central themes of [CollabW21](https://software.ac.uk/cw21), hosted online from 30th March to 1st April 2021, were FAIR Research Software, Diversity and Inclusion, and Software Sustainability.
+
+Some of the highlights included an inspiring [keynote on Diversity and Inclusion](https://youtu.be/8viA4y1pz_8?t=1943) from [Dr Chonnettia Jones](https://www.msfhr.org/chonnettia-jones) and an unbelievably brave panel discussion on [Disability and Accessibility in Research Software](https://www.youtube.com/watch?v=65a8c06VHOY).
```
Comment (MariaEriksson) [inline _posts/blog/2021-04-08-whitakerlab-at-collabw21.md:44]: Really like that you've included the youtube links here 💖

File: _posts/blog/2021-04-08-whitakerlab-at-collabw21.md [hunk 10]
```diff
@@ -0,0 +1,71 @@
...
+
+## The Lab's Reflections
+
+At the lab meeting after the event, each member that attended was asked to produce a slide reflecting on their experience at CollabW21.
+They were asked to highlight something they really enjoyed, was unexpected, or completely different to past conference experiences (signified by the :rocket: emoji); and a topic that resonated with them, got them thinking, that they’d like to share/discuss with the lab (signified by the :thinking: emoji).
```
Comment (MariaEriksson) [inline _posts/blog/2021-04-08-whitakerlab-at-collabw21.md]: I found this sentence a little difficult to follow. Could it perhaps be split into two sentences?

File: _posts/blog/2021-04-08-whitakerlab-at-collabw21.md [hunk 11]
```diff
@@ -0,0 +1,71 @@
...
+![](../../images/WhitakerLab_CollabW_ekaroune.jpg)
+
+![](../../images/WhitakerLab_CollabW_msharan.jpg)
+
+![](../../images/WhitakerLab_CollabW_pherterich.jpg)
+
+![](../../images/WhitakerLab_CollabW_amehonic.jpg)
+
+As you can see, themes around disability, diversity, inclusion and accessibility resonated particularly strongly within the group.
+But the unconference style felt very refreshing, if not a little exhausting!
```
Comment (MariaEriksson) [inline _posts/blog/2021-04-08-whitakerlab-at-collabw21.md]: I wasn't sure you needed the "But...". I may have written "The unconference style felt very refreshing, though some members found it a little exhausting!"

File: _posts/blog/2021-04-08-whitakerlab-at-collabw21.md [hunk 12]
```diff
@@ -0,0 +1,71 @@
...
+
+Additionally on the third day of CollabW, Sarah and Patricia's hackday projects on [Research Dugnads](https://research-dugnads.github.io/dugnads-hq/) and [Coding Confessions](https://coding-confessions.github.io/) won 3rd and 2nd place, respectively!
```
Comment (MariaEriksson) [inline _posts/blog/2021-04-08-whitakerlab-at-collabw21.md]:
```suggestion
Additionally on the third day of CollabW21, Sarah and Patricia's hackday projects on [Research Dugnads](https://research-dugnads.github.io/dugnads-hq/) and [Coding Confessions](https://coding-confessions.github.io/) won 3rd and 2nd place, respectively!
```

File: _posts/blog/2021-04-08-whitakerlab-at-collabw21.md [hunk 13]
```diff
@@ -0,0 +1,71 @@
...
+Well done Sarah and Patricia!
```
Comment (MariaEriksson) [inline _posts/blog/2021-04-08-whitakerlab-at-collabw21.md:74]: Wohooo 🎉

**Review by MariaEriksson — COMMENTED** [review]
Just a few minor suggestions 😌 Really enjoyed reading (and thanks for letting me practise reviewing on GitHub!)🌷

File: _posts/blog/2021-04-08-whitakerlab-at-collabw21.md (hunk 4 above)
Comment (sgibson91) [reply _posts/blog/2021-04-08-whitakerlab-at-collabw21.md]: Nope, I think that's an artefact from chopping and changing stuff in a previous review - good catch!

Comment (sgibson91) [reply _posts/blog/2021-04-08-whitakerlab-at-collabw21.md]:
```suggestion
The [Software Sustainability Institute](https://software.ac.uk/about) (SSI for short) is a network of UK universities, Research Software Engineering groups and policy makers dedicated to improving the quality, sustainability and recognition of research software.
```

File: _posts/blog/2021-04-08-whitakerlab-at-collabw21.md (hunk 11 above)
Comment (sgibson91) [reply _posts/blog/2021-04-08-whitakerlab-at-collabw21.md]: Could probably also say "We also found the unconference style..." ?

File: _posts/blog/2021-04-08-whitakerlab-at-collabw21.md (hunk 10 above)
Comment (sgibson91) [reply _posts/blog/2021-04-08-whitakerlab-at-collabw21.md]:
```suggestion
They were asked to answer the following questions:

 1. What is something you really enjoyed, was unexpected, or completely different to past conference experiences? (signified by the :rocket: emoji below)
 2. Can you share a topic that resonated with you, got you thinking, that you’d like to share/discuss with the lab? (signified by the :thinking: emoji below)
```

Does a list make it easier to read?

Comment (MariaEriksson) [reply _posts/blog/2021-04-08-whitakerlab-at-collabw21.md]: Yup! Love it 👍

File: _posts/blog/2021-04-08-whitakerlab-at-collabw21.md (hunk 11 above)
Comment (MariaEriksson) [reply _posts/blog/2021-04-08-whitakerlab-at-collabw21.md]: Yes, I like that!

**Review by MariaEriksson — APPROVED** [review]
Everything looks good to me! 👍🌻 Lovely blog post!

File: _posts/blog/2021-04-08-whitakerlab-at-collabw21.md (hunk 11 above)
Comment (sgibson91) [reply _posts/blog/2021-04-08-whitakerlab-at-collabw21.md]:
```suggestion
We also found the unconference style very refreshing, if not a little exhausting!
```

**Comment by sgibson91** [comment]
Woohoo! Thank you @MariaEriksson!! 🌻 🚀 ✨

**Pull request merged**